README.md

```diff
@@ -66,6 +66,8 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [nLocate](https://nlocate.com) | Harihar Bhawan, Lalitpur | Locate services, businesses, events around you
 
 ## P - R
+- [PageVamp](https://www.pagevamp.com/) | Sanepa, Lalitpur | Create and Update Website from Facebook Page
+- [PeaceNepal.com](http://peacenepal.com.np/) | Kandevtastan, Kupondole, Lalitpur | Website and Software Development, SEO, Web Hosting, Web Graphic and Logo Design
```

Comment:
Thanks @geshan dai. Can we narrow it down to `Kupondole, Lalitpur` to make it consistent.